src/ildasm/dis.cpp

```diff
@@ -1646,7 +1646,7 @@ BOOL Disassemble(IMDInternalImport *pImport, BYTE *ILHeader, void *GUICookie, md
 
             case ShortInlineBrTarget:
             {
-                char offset = (char) pCode[PC];
+                signed char offset = (signed char) pCode[PC];
```

Comment:
Yes, ARM assumes char unsigned: http://infocenter.arm.com/help/topic/com.arm.doc.faqs/ka11308.html

Because there might be other signedness sensitive char logic, I'll try `-fsigned-char` and replace the current commit with it if it does fine.